processor/resourcedetectionprocessor/internal/aws/lambda/config.go

```diff
@@ -0,0 +1,21 @@
+// Copyright The OpenTelemetry Authors
+// SPDX-License-Identifier: Apache-2.0
+
+package lambda // import "github.com/open-telemetry/opentelemetry-collector-contrib/processor/resourcedetectionprocessor/internal/aws/lambda"
+
+import (
+	"github.com/open-telemetry/opentelemetry-collector-contrib/processor/resourcedetectionprocessor/internal/aws/lambda/internal/metadata"
+)
+
+// Config defines user-specified configurations unique to the EC2 detector
+type Config struct {
+	// Tags is a list of regex's to match ec2 instance tag keys that users want
+	// to add as resource attributes to processed data
```

Comment:
same here and in other place

Reply:
Deleted everywhere

Reply:
This one it still there :)

Reply:
Thanks @mx-psi, right 🙈 Now it's gone for sure